lib/modulesync.rb

```diff
@@ -171,6 +171,7 @@ def self.update(options)
     # managed_modules is either an array or a hash
     managed_modules.each do |puppet_module, module_options|
       begin
+        puppet_module = puppet_module.to_s
```

Comment:
Can `managed_modules()` return puppet modules as strings? Or if `manage_module` needs strings, should that perform the cast?

Reply:
Actually, I don't think modules need to be strings, they can be managed internally as symbols I think.

Reply:
Going to restrict the call to `symbolize_keys` to fix that in a cleaner way.